src/index.css

```diff
@@ -56,6 +57,7 @@ html {
   --fs-mid-small-text: 1.8rem;
   --fs-small-text: 1.6rem;
   --fs-small-small-text: 1.4rem;
+  --fs-xsmall-text: 1.3rem;
```

Comment:
At some point, we should fixed this variables to be more representative to their name